package protoFile.pushOtherMessage;
message OtherMessage{
     required string msg = 1;
}